.editorconfig

```diff
@@ -0,0 +1,23 @@
+[*.{kt,kts}]
+end_of_line = lf
+ij_kotlin_allow_trailing_comma = false
+ij_kotlin_allow_trailing_comma_on_call_site = false
+ij_kotlin_imports_layout = *
+ij_kotlin_indent_before_arrow_on_new_line = false
+ij_kotlin_line_break_after_multiline_when_entry = true
+ij_kotlin_packages_to_use_import_on_demand = java.util.*,kotlinx.android.synthetic.**
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+ktlint_annotation_handle_annotations_with_parameters_same_as_annotations_without_parameters = [unset]
+                                                                                              ktlint_argument_list_wrapping_ignore_when_parameter_count_greater_or_equal_than = 8
+                                                                                              ktlint_chain_method_rule_force_multiline_when_chain_operator_count_greater_or_equal_than = 4
+                                                                                              ktlint_class_signature_rule_force_multiline_when_parameter_count_greater_or_equal_than = unset
+                                                                                              ktlint_code_style = android_studio
+                                                                                              ktlint_enum_entry_name_casing = upper_or_camel_cases
+                                                                                              ktlint_function_naming_ignore_when_annotated_with = [unset]
+                                                                                                                                                  ktlint_function_signature_body_expression_wrapping = default
+                                                                                                                                                  ktlint_function_signature_rule_force_multiline_when_parameter_count_greater_or_equal_than = unset
+                                                                                                                                                  ktlint_ignore_back_ticked_identifier = false
+                                                                                                                                                  ktlint_property_naming_constant_naming = screaming_snake_case
+                                                                                                                                                  max_line_length = 100
```

Comment:
The spacing here looks cursed.

Reply:
I agree, I pasted the configuration from ktlint generator and android studio formatted it to this.